queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
```

Comment:
- [ ] You've added queries for `List the date of birth of all animals named either 'Agumon' or 'Pikachu'` and `List name and escape attempts of animals that weigh more than 10.5kg`. It's a good addition to your project, but you should also include a screenshot of the results for these two queries in the PR description, which is currently missing. Please consider to add those screenshots.

Keep up the good work, and these adjustments will make your project even better! 👍📷👨‍💻